README.md

```diff
@@ -18,9 +18,10 @@ Code:
 ```
 //Request MODE 1, PID 0C - RPM
 val pid = PID(ObdModes.MODE_01, "0C")
+pid.formula = "(256A+B)/4" // Engine RPM formula https://en.wikipedia.org/wikiOBD-II_PIDs#Service_01

```

Comment:
You dont have to build this PID yourself.

You can request a fully built pid from the built in library this way:
```
val pid = PIDUtils.getPid(ObdModes.MODE_01, "OC")
```